src/Notes.cc

```diff
@@ -354,6 +354,15 @@ NotePairs::appendNewOnly(const NotePairs *src)
     }
 }
 
+void
+NotePairs::replaceOrAddOrAppend(const NotePairs *src, const std::vector<SBuf> &appendKeys)
+{
+    for (auto e: src->entries)
```

Comment:
```suggestion
    for (const auto e: src->entries) {
```

Reply:
I will change this to "const" but please note that there are other similar functions which dont use "const". (although I will not make changes there unless you tell me to do so)

Reply:
Done.

Reply:
Thank you. New and changed code should use const whenever possible. Old code should not be changed just for the sake of adding the missing const.